### ISSUE TITLE
ScalaJS support

### ISSUE THREAD

**Issue by mbaechler** [circe/circe-yaml#285]
As far as I understand, circe-yaml is not published as a ScalaJs lib.

Is there something I miss? If not, is there something preventing compilation into ScalaJs?

Thanks for your answer.

**Comment by armanbilge** [comment]
> If not, is there something preventing compilation into ScalaJs?

Yes, this library depends on the Java Snake Yaml library, which isn't available for Scala.js (since it's written in Java).

Until recently, there weren't any YAML parsers written in pure Scala AFAIK. But now there is one from VirtusLab, so I created a fork of this project that uses it instead and cross-publishes for Scala.js

https://github.com/armanbilge/circe-scala-yaml/releases/tag/v0.0.1

**Comment by mbaechler** [comment]
Thanks very much for the link and the project, it looks really cool.
